### ISSUE TITLE
Pain on thumb extension

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10756]
**Preferred term label:**
Pain on thumb extension

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Extensor pollicus longus (EPL) tendonitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001303
- label: Pain on thumb extension
- parents: [Hand pain (HP:0046505)](https://hpo.jax.org/app/browse/term/HP:0046505)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort that is triggered upon extension (straightening) of the thumb.
- comment: Pain on thumb extension may be observed in extensor pollicus longus (EPL) tendonitis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10756
- synonyms:
